tools/build_defs/configure_script.bzl

```diff
@@ -34,27 +35,50 @@ def create_configure_script(
     )]
     return "\n".join(script)
 
-def _define_deps_flags(inputs):
-    libs = collect_libs(inputs.deps_linking_info)
-
+def _define_deps_flags(deps, inputs):
```

Comment:
What is the type of inputs? I can't seem to trace it. And did it change in this change?

Reply:
Yep, this comes from the commits I mentioned earlier: https://github.com/bazelbuild/rules_foreign_cc/commit/d4c55bb51e70c736658d3afc8365575f49471bfe
I will add the documentation.
Thank you for a catch!

Reply:
The improvemnts for the naming and documentation go here: https://github.com/bazelbuild/rules_foreign_cc/pull/116
The pull request is based on this one.